tour/ranges-splits.md

```diff
@@ -32,8 +32,8 @@ Take a minute to learn these Accumulo terms:
 
 Knowing these terms are critical when working closely with Accumulo.  Iterators are especially unique and powerful.  More on them later.
 
-When working with large amounts of data across many Tablet Servers, a simple Scanner might not do the trick. Next lesson we learn about the power of 
-the multi-threaded BatchScanner!  
+When working with large amounts of data across many `Tablet Servers`, a simple `Scanner` might not do the trick. Next lesson we learn about the power of 
```

Comment:
"Tablet Servers" isn't a class name. If your intent is to highlight only class names, then we could highlight it as `TabletServer`s, but in this case, I don't think it should be highlighted at all, because it makes sense to use the semi-proper name "Tablet Server" rather than the class name, "TabletServer".

```suggestion
When working with large amounts of data across many Tablet Servers, a simple `Scanner` might not do the trick. Next lesson we learn about the power of 
```

Reply:
Yeah I originally thought you wanted to highlight all the class names. Looking back now, I agree that we should get rid of the redundant highlighted words as. I will go through the tour code and get rid of some of the highlighted words.